WebRoot/skins/_base/base4/skin.properties

```diff
@@ -1773,7 +1778,7 @@ CheckboxTableRightText      = text-align:left; margin-left:@CheckBoxTableCellSpa
 MailAttachmentsTable        = @FullWidth@
 MailAttachmentItem          = @FullWidth@ @AppBg@ margin-bottom:2px; @roundCorners(3px)@ overflow:hidden;
 MailAttachmentItemActive    = background-color: @darken(AppC,13)@;
-MailAttachmentIcon          = @BigIconSize@
+MailAttachmentIcon          = @BigIconSize@; @roundCorners(0px)@ !important;
```

Comment:
We required this to override the generic styling for roundCorners coming from "FileIconCommonStyle" . We need no roundCorners only in mail reading view. 

@kalpeshhpatel  can also help on this maybe.